client/src/services/DonuService.ts

```diff
@@ -12,19 +13,85 @@ const callDonu = (request: Donu.Request): Promise<Donu.Response> => {
   }
 };
 
+export const getDonuModelSource = async (model: string, type: Donu.Type): Promise<Donu.ModelGraph> => {
```

Comment:
Does this function needs to be exported?

Reply:
its called strictly within the function true, we can remove the exported and add it back if its ever needed later